nncf/tensorflow/quantization/algorithm.py

```diff
@@ -451,11 +463,12 @@ def _get_quantizable_weighted_layer_nodes(self, nncf_graph: NNCFGraph) -> List[Q
 
             assert issubclass(metatype, TFLayerWithWeightsMetatype)
             nodes_with_weights.append(node)
+        scope_overrides_dict = self._get_algo_specific_config_section().get('scope_overrides', {})
         weighted_node_and_qconf_lists = assign_qconfig_lists_to_modules(nodes_with_weights,
                                                                         self.DEFAULT_QCONFIG,
                                                                         self.global_quantizer_constraints[
                                                                             QuantizerGroup.WEIGHTS],
-                                                                        scope_overrides_dict=None,
+                                                                        scope_overrides_dict,
```

Comment:
Just notice, it looks like `scope_overrides` didn`t work for weights before in TF